internal/models/pubkey_resource_test.go

```diff
@@ -0,0 +1,15 @@
+package models
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTagFormat(t *testing.T) {
+
+	pubkey := PubkeyResource{12, "randomtag", 25, 1, "aws-handle"}
+
+	// assert equality
+	assert.Equal(t, "pk-12-randomtag", pubkey.FormattedTag(), "the tag format is incorrect")
+}
```

Comment:
This is a good unit test candidate indeed 👍🏻

Reply:
Note we don't need to test edge cases because database ensures that both PK and Tag are never nil or blank. That is going to change today, but I will add an extra test for this.